devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy

```diff
@@ -177,8 +191,18 @@ def refresh() {
 }
 
 def configure() {
-	// Device-Watch allows 2 check-in misses from device + ping (plus 2 min lag time)
-	sendEvent(name: "checkInterval", value: 2 * 10 * 60 + 2 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
-	log.debug "Configuring Reporting and Bindings."
-	zigbee.onOffRefresh() + zigbee.onOffConfig()
-}
\ No newline at end of file
+	def manufacturer = getDataValue("manufacturer")
+
+	if (manufacturer == "Third Reality, Inc") {
+		def enrollCmds = zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0021) + zigbee.readAttribute(zigbee.IAS_ZONE_CLUSTER,zigbee.ATTRIBUTE_IAS_ZONE_STATUS)
+		return zigbee.addBinding(zigbee.IAS_ZONE_CLUSTER) + enrollCmds
```

Comment:
do you not need to add bindings for the onOff cluster?

Also, based on your original commit, you do not follow the zigbee standard of reporting the remaining battery percentage doubled, so you would need to add that back in